packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/DeprecatedClass.java

```diff
@@ -0,0 +1,65 @@
+package software.amazon.jsii.tests.calculator;
+
+/**
+ * This tests code generation of deprecation markers.
+ * 
+ * @deprecated without replacement
```

Comment:
Where is @stability?

Reply:
I find it silly to tag both `@stability deprecated` and `@deprecated` (and also `@Deprecated`). This is the "Java-canonical" way to denote deprecation.

Reply:
I've worked around this itch in a clever (very java-y) way :P